examples/node/scripts/reactions/addReaction.ts

```diff
@@ -0,0 +1,33 @@
+import { LensClient, PublicationReactionType, development } from "@lens-protocol/client";
```

Comment:
filenames should be the same as method name, so /scripts/publication/reactions/ and then
- add.ts
- remove.ts
- fetch.ts

Reply:
I see, then up to you but please nest it under /publication/ folder to keep the structure

Reply:
done 👍